packages/vue/src/useUploadThing.ts

```diff
@@ -129,8 +126,15 @@ export const INTERNAL_uploadthingHookGen = <
     return {
       startUpload,
       isUploading,
-      permittedFileInfo,
-    } as const;
+      routeConfig,
+      /**
+       * @deprecated Use `routeConfig` instead
+       */
+      permittedFileInfo: computed(() => {
+        if (!routeConfig) return undefined;
```

Comment:
is this not a ref so this will always be defined? should be `routeConfig.value`?

```suggestion
        if (!routeConfig.value) return undefined;
```